FindChannelFunction/src/main/java/no/unit/nva/channel/GatewayResponse.java

```diff
@@ -0,0 +1,45 @@
+package no.unit.nva.channel;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GatewayResponse<T> {
+
+    private T body;
+    private Map<String, String> headers;
+    private int statusCode;
+
+    public GatewayResponse() {
```

Comment:
Unused constructors and methods, but I guess you have your reasons

Reply:
Jeg har fjernet en del default constructors og setters og erstattet med JsonCreator, men getterne brukes fortsatt av jackson ved mapping.